### PR TITLE
adding material to begginner path

### PR DESCRIPTION
**Pull request by DaleR-Harrison**
## Related Issue 
No related issue (will make sure to creat one first going forward)

- Information about the related issue 

Closes: # 

## Type of change

Changes adds a short written path at beginning pf beginner dev path. Plan to extend this in future to give info on creating dev environment. Also adds a new section which has some recommended reading. 

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Not a code change

## Checklist

- [x] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:27]: Missing an extra line here (Markdown linter stuff).
```suggestion
## Books

```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:5]:
```suggestion
To develop decentralized applications on Ethereum or other blockchains, you will need some fundamental skills. You may already have some of these skills. If so, that's great! If you are here though, at least some of this is likely new to you. Here are the essentials of what you need to know.
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:9]:
```suggestion
You will need to have a basic understanding of how the blockchain works. What is private and public key? What is a block and how is it validated? What even is a blockchain. There are some great materials out there for you and we have gathered some of the best below. If books are your thing, you'll probably want to start with "Mastering Bitcoin." This will give you some interesting history and teach you everything you need to know about Bitcoin.
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:11]:
```suggestion
You can then follow this up with "Mastering Ethereum". As a Dapp developer specifically, you will need to know about smart contracts and the EVM. This will not only teach you about Ethereum but also many of the concepts you will need for developing on other chains. 
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:15]:
```suggestion
The "backend" of your decentralized applications will be smart contracts that are deployed onto a blockchain. While there are a few other languages that are being experimented with or are in limited use, Solidity is the go-to language for developing on Ethereum and EVM compatible blockchains. In the training section of this dev-path, there are some excellent tutorials to get you started. Cryptozombies in particular is one of the best introductions to Solidity basics out there.
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:21]:
```suggestion
While the blockchain can function as your backend, you will need to provide users of your app with a way to interact with your contracts. Python and Javascript are the two primary scripting languages used currently for Dapp development. Excellent libraries have been developed for both which allow interaction with the blockchain. You will also need to know some HTML and CSS to build the UI for your Dapp.  
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:23]:
```suggestion
    This is a little outside the context of this guide. Luckily there are many resources out there to help you learn. Our partner community, Dev.to, is full of amazing resources and a great community as well. If you have a little knowledge already, or want to learn as you go, many of the tutorials linked in our training section also provide some practice with frontend languages. 
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:25]:
```suggestion
The above may not seem like much, but it is all you need to join the revolution in web development that is Web3. Once you've gotten some basic experience with the above and built a few simple decentralized applications of your own, you'll be ready to move on to our intermediate and advanced blockchain developer guides. 
```

File: learning-guide/beginner.md
Comment (Panquesito7) [inline learning-guide/beginner.md:31]:
```suggestion
| **Mastering Ethereum** | Mastering Ethereum is a book for developers, offering a guide to the operation and use of the Ethereum, Ethereum Classic, RootStock (RSK), and other compatible EVM-based open blockchains. | https://github.com/ethereumbook/ethereumbook
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Awesome, thanks! 🚀

**Comment by Panquesito7** [comment]
Next time, please don't merge it until 2 or more maintainers have approved. Thanks. 🙂

**Review by krishguptadev — APPROVED** [review]

**Pull request merged**
